test/unit/org/apache/cassandra/security/PEMBasedSslContextFactoryTest.java

```diff
@@ -215,13 +215,14 @@ public void getSslContextOpenSSL() throws IOException
     {
         ParameterizedClass sslContextFactory = new ParameterizedClass(PEMBasedSslContextFactory.class.getSimpleName()
         , new HashMap<>());
-        EncryptionOptions options = new EncryptionOptions.Builder().withTrustStore(TlsTestUtils.SERVER_TRUSTSTORE_PEM_PATH)
-                                                                   .withKeyStore(TlsTestUtils.SERVER_KEYSTORE_PATH_PEM)
-                                                                   .withKeyStorePassword(TlsTestUtils.SERVER_KEYSTORE_PASSWORD)
-                                                                   .withRequireClientAuth(NOT_REQUIRED)
-                                                                   .withCipherSuites("TLS_RSA_WITH_AES_128_CBC_SHA")
-                                                                   .withSslContextFactory(sslContextFactory)
-                                                                   .build();
+        EncryptionOptions options = new EncryptionOptions.ServerEncryptionOptions.Builder()
```

Comment:
@smiklosovic originally this was testing for `ClientEncryptionOptions`. As you see this doesn't have any server specific tests like outbound keystores and internode encryption so we should change this back to `ClientEncryptionOptions` here in my opinion.